setup/github-bot/ansible-playbook.yaml

```diff
@@ -0,0 +1,73 @@
+---
+- hosts: github-bot
+  remote_user: root
+  gather_facts: False
+
+  tasks:
+
+    - name: Bootstrap | Install baseline packages
+      raw: apt-get install -y aptitude python-apt apt-transport-https
+      tags: bootstrap
+
+- hosts: github-bot
+  remote_user: root
+
+  tasks:
+
+    - include_vars: ansible-vars.yaml
+      tags: vars
+
+    - name: General | Add the NodeSource GPG key
+      apt_key: url=https://deb.nodesource.com/gpgkey/nodesource.gpg.key state=present
+      tags: general
+
+    - name: General | Add the NodeSource Node.js repo
+      apt_repository: repo='deb https://deb.nodesource.com/node_6.x jessie main' state=present
+      tags: general
+
+    - name: General | APT Update and upgrade
+      apt: update_cache=yes upgrade=full
+      tags: general
+
+    - name: General | Install required packages
+      apt: name="{{ item }}" state=latest
+      with_items: "{{ packages }}"
+      tags: general
+
+    - name: General | Enable time syncing
+      service: name=systemd-timesyncd state=started enabled=yes
+      tags: general
+
+    - name: User | Add {{ server_user }} user
+      user: name="{{ server_user }}" shell=/bin/bash
+      tags: user
+
+    - name: Init | Generate and copy init script
+      template: src=./resources/github-bot.service dest=/lib/systemd/system/github-bot.service
+      tags: init
+
+    - name: Init | Create directory for EnvironmentFile
+      file: path=/home/"{{ server_user }}"/environment state=directory mode=0755 owner="{{ server_user }}"
+      tags: init
+
+    - name: Init | Generate and copy systemd EnvironmentFile
+      template: src=./resources/environment-file dest=/home/"{{ server_user }}"/environment/github-bot
+      tags: init
+
+    - name: Init | Clone github-bot repo
+      become: yes
+      become_user: "{{ server_user }}"
+      git: repo=https://github.com/nodejs/github-bot.git dest=/home/"{{ server_user }}"/github-bot
+      tags: init
+
+    - name: Init | Install npm dependencies
+      become: yes
+      become_user: "{{ server_user }}"
+      command: "npm install"
```

Comment:
You can use the [npm module](http://docs.ansible.com/ansible/npm_module.html) here.